src/main/webapp/adminOnly/AdminOnly.js

```diff
@@ -106,9 +111,8 @@ const AdminOnly = hh(class AdminOnly extends Component {
   }
 
   parseDate = (date) => {
-    if (date !== null) {
-      let d = new Date(date).toISOString();
-      return d.slice(0, d.indexOf("T"));
+    if (date != null) {
```

Comment:
Lodash has an [`isDate`](https://lodash.com/docs/2.4.2#isDate) function as does [moment](https://momentjscom.readthedocs.io/en/latest/moment/05-query/11-is-a-date/) but neither work on plain strings :-(